### DISCUSSION TITLE
After updating OnionOS to v4.3.1-1, RetroArch has crapped the bed.

### DISCUSSION THREAD

**Discussion by LordGideon** [OnionUI/Onion#1672] [Support] [answered]
OnionOS seems to be unaffected, but as the title states - any game I try to load now has a scrambled screen on it. The only way to back out (as the screen is a mess) is to yank the battery, then hold down the menu button while the device boots to get it to go back to the main menu and _not_ resume where it left off. 

Any ideas?

![1](https://github.com/user-attachments/assets/3efb086a-fbb5-4aac-a85d-3650e6f63e8b)
![2](https://github.com/user-attachments/assets/0f5943d8-75a5-495a-a08e-11e91321cb38)

**Comment by LordGideon** [comment, marked as answer]
Replacing the entire RetroArch folder with a backup of the same folder from before I updated OnionOS solved the problem.

**Comment by Nighted** [comment]
I had an issue also when I updated and had to move the retroarch.ini back to the appropriate directory because it had been moved during the process. Back to normal after that. My screen wasn't like that though.
